kickstarts/almalinux-8-base.x86_64.ks

```diff
@@ -0,0 +1,92 @@
+# AlmaLinux 8 kickstart file for x86_64 base Docker image
+
+# install
+url --url https://repo.almalinux.org/almalinux/8/BaseOS/x86_64/os/
+
+lang en_US.UTF-8
+keyboard us
+timezone --nontp --utc UTC
+
+network --activate --bootproto=dhcp --device=link --onboot=on
+selinux --disabled
+
+bootloader --disable
+zerombr
+clearpart --all --initlabel
+autopart --fstype=ext4 --type=plain --nohome --noboot --noswap
+
+rootpw --iscrypted --lock almalinux
+
+shutdown
+
+%packages --excludedocs --nocore --instLangs=en --excludeWeakdeps
+almalinux-release
+bash
+coreutils-single
+crypto-policies-scripts
+findutils
+gdb-gdbserver
+glibc-minimal-langpack
+gzip
+libuser
+passwd
+rootfiles
+systemd
+tar
+usermode
+vim-minimal
+virt-what
+which
+yum
+-dnf-plugin-subscription-manager
+-dosfstools
+-e2fsprogs
+-fuse-libs
+-gnupg2-smime
+-kernel
+-libss
+-open-vm-tools
+-pinentry
+-qemu-guest-agent
+-shared-mime-info
+-subscription-manager
+-trousers
+-xfsprogs
+-xkeyboard-config
+%end
+
+
+# NOTE: add --log=/root/anaconda-post.log for debugging
+%post --erroronfail
+# generate build time file for compatibility with CentOS
+/bin/date +%Y%m%d_%H%M > /etc/BUILDTIME
+
+# set DNF infra variable to container for compatibility with CentOS
+echo 'container' > /etc/dnf/vars/infra
+
+# import AlmaLinux PGP key
+rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux
+
+# install only en_US.UTF-8 locale files, see
+# https://fedoraproject.org/wiki/Changes/Glibc_locale_subpackaging for details
+echo '%_install_langs en_US.UTF-8' > /etc/rpm/macros.image-language-conf
+
+# force each container to have a unique machine-id
+> /etc/machine-id
+
+# create tmp directories because there is no tmpfs support in Docker
+umount /run
+systemd-tmpfiles --create --boot
+
```

Comment:
I believe following lines may needs to be added.

```sh
# disable login prompt and mounts
systemctl mask console-getty.service \
               dev-hugepages.mount \
               getty.target \
               systemd-logind.service \
               sys-fs-fuse-connections.mount \
               systemd-remount-fs.service
```

Reply:
if systemd not starting when container started, how this lines will work? 🤔

Reply:
These are definitively required. It creates link to `/dev/null` entries in `/etc/systemd/system` folder.

`redhat/ubi8` 
```sh
lrwxrwxrwx 1 root root    9 Aug  3 16:25 console-getty.service -> /dev/null
lrwxrwxrwx 1 root root   41 Aug  3 16:25 default.target -> /usr/lib/systemd/system/multi-user.target
lrwxrwxrwx 1 root root    9 Aug  3 16:25 dev-hugepages.mount -> /dev/null
lrwxrwxrwx 1 root root    9 Aug  3 16:25 getty.target -> /dev/null
drwxr-xr-x 2 root root 4096 Aug  3 16:25 getty.target.wants
drwxr-xr-x 2 root root 4096 Aug  3 16:25 multi-user.target.wants
lrwxrwxrwx 1 root root    9 Aug  3 16:25 sys-fs-fuse-connections.mount -> /dev/null
lrwxrwxrwx 1 root root   39 Aug  3 16:25 syslog.service -> /usr/lib/systemd/system/rsyslog.service
lrwxrwxrwx 1 root root    9 Aug  3 16:25 systemd-logind.service -> /dev/null
lrwxrwxrwx 1 root root    9 Aug  3 16:25 systemd-remount-fs.service -> /dev/null
drwxr-xr-x 2 root root 4096 Aug  3 16:25 timers.target.wants
```

on the new system
```sh
lrwxrwxrwx 1 root root   41 Sep  9 21:36 default.target -> /usr/lib/systemd/system/multi-user.target
drwxr-xr-x 2 root root 4096 Sep  9 21:36 getty.target.wants
drwxr-xr-x 2 root root 4096 Sep  9 21:36 multi-user.target.wants
lrwxrwxrwx 1 root root   39 Sep  9 21:36 syslog.service -> /usr/lib/systemd/system/rsyslog.service
drwxr-xr-x 2 root root 4096 Sep  9 21:36 timers.target.wants
```